crates/hir-def/src/nameres/collector.rs

```diff
@@ -244,7 +246,7 @@ struct DefCollector<'a> {
     /// This also stores the attributes to skip when we resolve derive helpers and non-macro
     /// non-builtin attributes in general.
     // FIXME: There has to be a better way to do this
-    skip_attrs: FxHashMap<InFile<ModItem>, AttrId>,
+    skip_attrs: FxHashMap<InFile<FileAstId<ast::Item>>, AttrId>,
```

Comment:
Good catch! It's indeed better to store an ast id here, for stability of the `DefMap`.